docs/section10/10-5.md

```diff
@@ -0,0 +1,35 @@
+# Airside systems – Rule 10-5  
+**Schema Version:** 0.0.12  
+**Mandatory Rule:** False   
+**Rule ID:** 10-5  
+ 
+**Rule Description:** For HVAC systems designed, mechanical equipment cooling efficiencies shall be adjusted from actual design conditions to the standard rating conditions.   
```

Comment:
The corresponding rule in appendix G says that it should only be done "... if required by the simulation model". Please add this to the description because I think this is an important detail. I also updated the language in the Rules spreadsheet accordingly.

Reply:
Got it. Done.